File: Civitas ruby/lib/juego_texto.rb
require_relative "vista_textual"
require_relative "civitas_juego"
require_relative "controlador"

module Civitas
  class Juego_texto

    def self.main   
      nombres = []
        
      nombres << "Javi"
      nombres << "Pedro"
      nombres << "Antonio"
    
      vista = Vista_textual.new
      civitas = Civitas_juego.new(nombres)
      controlador = Controlador.new(civitas, vista)
      controlador.juega
    end
  end
  
  Juego_texto.main
end

File: Civitas java/src/juegoTexto/juegoTexto.java
package juegoTexto;

import civitas.CivitasJuego;
import civitas.Dado;
import java.util.ArrayList;

public class juegoTexto {
    public static void main(String[] args){
        VistaTextual vista = new VistaTextual();
        ArrayList<String> nombres = new ArrayList();
        Dado.getInstance().setDebug(false);
        
        nombres.add("Javi");
        nombres.add("Antonio");
        nombres.add("Pedro");
        
        CivitasJuego civitas = new CivitasJuego(nombres);
        
        Controlador controlador = new Controlador(civitas, vista);
        
        controlador.juega();
    }
}
